addon-test-support/scroll-into-view-mock.js

```diff
@@ -0,0 +1,31 @@
+import sinon from 'sinon';
+import { find } from '@ember/test-helpers';
+export default function setupScrollIntoView(hooks) {
+  hooks.beforeEach(function () {
+    this.scrollIntoViewSandbox = sinon.createSandbox();
```

Comment:
We don't need to use `createSandbox`. Sinon is its own sandbox now, and cleanup happens automatically if we use `ember-sinon-qunit`.

https://sinonjs.org/releases/latest/sandbox/
https://github.com/elwayman02/ember-sinon-qunit

We already have the integration set up, so you can just call `sinon.stub` directly and don't need to use the `afterEach` hook for cleanup - it happens automatically.

https://github.com/elwayman02/ember-scroll-modifiers/blob/master/tests/test-helper.js#L13